CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Improve code-behind feature file compilation speed (#336)
 * Improve parameter type naming for generic types (#343)
 * Reduced MsBuild log output and consistent use of [Reqnroll] prefix (#381)
+* Update behavior of `ObjectContainer.IsRegistered()` to check base container for registrations, to match `Resolve()` behavior (#367)
```

Comment:
Given this is a "breaking"/behavior change, is there any major/minor version specification I should be revving as part of this PR? Or will maintainers handle that as part of the next release?

Is this message clear enough to indicate it's a behavior change? I see historical change entries for bug fixes have a `Fix: ` prefix, and new features/support seem to not have anything. Should there be a `Breaking: ` (or similar) prefix to this one?  (It could be argued it's a `Fix` for behavior it should have always had, or a "design bug fix", but it's not _really_ a code bug.)


If you'd like any of the above☝️ concepts solidified, let me know and I can create another PR to update the `CONTRIBUTING.md`.